### PR TITLE
update contributors for onnx repo

### PR DESCRIPTION
**Pull request by liqunfu**
Signed-off-by: Liqun Fu <liqfu@microsoft.com>

### CODE REVIEW

File: CONTRIBUTORS [hunk 1]
```diff
@@ -42,13 +55,21 @@ SIG-archinfra:
 
 
 SIG-operators:
+    - ashbhandare
     - bowenbao
     - daquexian
     - garymm
     - hariharans29
+    - hwangdeyu
+    - jiafatom
```
Comment (garymm) [inline CONTRIBUTORS:64]: I don't expect them to continue to contribute

File: CONTRIBUTORS [hunk 2]
```diff
@@ -42,13 +55,21 @@ SIG-archinfra:
...
+    - liuyu21
     - matteosal
+    - neginraoof
```
Comment (garymm) [inline CONTRIBUTORS:67]: I don't expect them to continue to contribute

File: CONTRIBUTORS [hunk 3]
```diff
@@ -10,14 +10,27 @@
...
+    - jantonguirao
     - kevinch-nv
+    - mhamilton723
+    - rajeevsrao
     - skottmckay
+    - snnn
     - spandantiwari
+    - stillmatic
+    - take-cheeze
+    - TomWildenhain-Microsoft
```
Comment (garymm) [inline CONTRIBUTORS:29]: I don't expect them to continue to contribute

**Review by garymm — COMMENTED** [review]
Does the lack of removals mean that all current contributors have contributed in the past 12 months?

If not, can we also remove inactive people in this PR?

**Review by jcwchen — COMMENTED** [review]
Other than adding more contributors, to cleanup, I think it's also a good time to review existing contributors and remove some of them if they are inactive for a while.

File: CONTRIBUTORS (hunk 1 above)
Comment (liqunfu) [reply CONTRIBUTORS]: updated to not adding them.

File: CONTRIBUTORS (hunk 3 above)
Comment (liqunfu) [reply CONTRIBUTORS]: updated to not adding them.

File: CONTRIBUTORS (hunk 2 above)
Comment (liqunfu) [reply CONTRIBUTORS]: updated to not adding neginraoof.

**Review by AlexandreEichenberger — COMMENTED** [review]
Does not appear to be sorted alphabetically

File: CONTRIBUTORS [hunk 4]
```diff
@@ -10,15 +10,22 @@
...
+    - mhamilton723
+    - rajeevsrao
     - skottmckay
-    - spandantiwari
-    - vinitra
-    - winnietsang
+    - snnn
     - wschin
     - yuanbyu
+    - xuzijian629
```
Comment (jcwchen) [inline CONTRIBUTORS:28]:
```suggestion
    - xuzijian629
    - yuanbyu
    - yuslepukhin
```

**Review by AlexandreEichenberger — APPROVED** [review]
LGTM, not directly involved with arch infra and operator so cannot vouch directly for the involvement of the folks listed here.

**Review by linkerzhang — APPROVED** [review]
LGTM.

**Pull request merged**